src/pages/sign-in.tsx

```diff
@@ -13,6 +13,8 @@ import {
 } from '~/features/sign-in/components'
 import { withSessionSsr } from '~/lib/withSession'
 
+const title = 'Starter Kit'
```

Comment:
Probably a good TODO to set an APP_NAME env var

Reply:
I've done so... users can now set `NEXT_PUBLIC_APP_NAME` if they want to override the application name